rflx/specification/parser.py

```diff
@@ -1134,10 +1135,7 @@ def create_refinement(
     )
 
 
-def check_naming(
-    error: RecordFluxError, package: PackageNode, _filename: Path, origname: Path = None
-) -> None:
-    name = origname or Path("<stdin>")
+def check_naming(error: RecordFluxError, package: PackageNode, _filename: Path, name: Path) -> None:
```

Comment:
What is the difference between `_filename` and `name`? These names are not self-explanatory. Actually, the parameter `_filename` seems very useless.

Reply:
It's indeed useless and removed in 66eeeb290cbcd5feebee3eba69f8c9da3e94b81d.